pytripgui/model/plot_model.py

```diff
@@ -10,38 +10,82 @@
 
 class ProjectionSelector:
     def __init__(self):
-        self.current_x_slice = 50
-        self.current_y_slice = 125
-        self.current_z_slice = 125
+        self.current_T_slice = 0
+        self.current_S_slice = 0
+        self.current_C_slice = 0
+
+        self.max_T_depth = 0
+        self.max_S_depth = 0
+        self.max_C_depth = 0
 
         # "Transversal" (xy)
         # "Sagittal" (yz)
         # "Coronal"  (xz)
         self.plane = "Transversal"
 
     def next_slice(self):
+        if self.position < self._max_position():
+            self.position += 1
+        else:
+            self.position = 0
+
+    def prev_slice(self):
+        if self.position != 0:
+            self.position -= 1
+        else:
+            self.position = self._max_position()
+
+    def get_projection(self, data):
+        if self.plane == "Transversal":
+            return data.cube[self.position]
+        elif self.plane == "Sagittal":
+            return data.cube[-1:0:-1, -1:0:-1, self.position]
+        elif self.plane == "Coronal":
+            return data.cube[-1:0:-1, self.position, -1:0:-1]
+
+    def load_slices_count(self, data):
+        self.max_T_depth = data.dimz - 1
+        self.max_S_depth = data.dimy - 1
+        self.max_C_depth = data.dimx - 1
+
+        self.current_T_slice = int(self.max_T_depth / 2)
```

Comment:
Why not to use integer division like `2 // 3`

Reply:
ok